src/site/apt/index.apt.vm

```diff
@@ -111,6 +113,17 @@ java -jar apache-rat-${project.version}.jar --help
 +------------------------------------------+
 
  Read more {{{./apache-rat-plugin/index.html} here}}.
+ 
+ ** Styling output
+ 
+ Rat allows you to style the output as you see fit.  Three stylesheets are included in the rat package.
+ 
+    plain-rat  The default stylesheet.
```

Comment:
Wouldn't it make sense to link to the stylesheets from the webpage? So people could directly take a look?

Reply:
fyi: I'll will try to add a link on the the branch ....

Reply:
Will commit directly to your branch - hope this is okay,  @Claudenw 
fyi:
https://maven.apache.org/doxia/references/apt-format.html